ccp/modules/dnpm-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: "3.7"
-
```

Comment:
Why is this removed?

Reply:
It is already declared in main docker-compose.yml